awx/main/tests/functional/test_copy.py

```diff
@@ -20,10 +20,10 @@ def test_job_template_copy(post, get, project, inventory, machine_credential, va
     job_template_with_survey_passwords.admin_role.members.add(alice)
     project.admin_role.members.add(alice)
     inventory.admin_role.members.add(alice)
-    assert get(
+    get(
         reverse('api:job_template_copy', kwargs={'pk': job_template_with_survey_passwords.pk}),
-        alice, expect=200
-    ).data['can_copy'] is False
+        alice, expect=403
```

Comment:
oh, I think I see what happened. For the get request, a 200 would be preferable for alice, who has admin for the job template.

I think we just want to catch `PermmissionDenied` in

https://github.com/ansible/awx/blob/cf2ad1486a293906479961696b623086bccc53f7/awx/api/generics.py#L963

And give `False` for the value in that case.